birbal/so_user_info_cache.py

```diff
@@ -0,0 +1,31 @@
+import sqlite3
+import json
+
+class SqliteHandler:
+    def __init__(self, path, logger):
+        self.logger = logger
+        self.logger.debug("Connecting the table %s", path)
+        self.db = sqlite3.connect(path)
+        self.cu = self.db.cursor()
+        self.create_table()
+
+    def create_table(self):
+        self.cu.execute('''
+        CREATE TABLE IF NOT EXISTS users(id INTEGER PRIMARY KEY, updated_at BIGINT, userdata TEXT,
+        userid BIGINT)''')
+        self.db.commit()
+        
+    def insert_user(self, time, userid, userdata):
+        self.cu.execute('''INSERT INTO users(updated_at, userdata, userid)
+                  VALUES(?,?,?)''', (time, json.dumps(userdata), userid))
```

Comment:
I'm guessing users table is about SO users and storing their reputation. Perhaps later we can store only the reputation instead of storing the JSON dump, which may be quite big.